cmd/tckconvert.cpp

```diff
@@ -89,8 +91,12 @@ void usage ()
 
   + OptionGroup ("Options for both PLY and RIB writer")
 
-  + Option ("radius", "radius of the streamlines")
-  +   Argument("radius").type_float(0.0f);
+    + Option ("radius", "radius of the streamlines")
+    +   Argument("radius").type_float(0.0f)
+
+    + OptionGroup ("Options VTK writer")
```

Comment:
Could we make that "Options **specific to** VTK writer" for consistency with the other sections...?

Reply:
Done.